package.json

```diff
@@ -16,16 +16,15 @@
   },
   "devDependencies": {
     "@vuepress/bundler-vite": "2.0.0-rc.18",
-    "@vuepress/bundler-webpack": "2.0.0-rc.17",
```

Comment:
Forgot to make this a separate commit, but I got rid of the webpack bundler since we were using vite to build the site and removing it didn't seem to break anything that I saw.